beta/nncf/tensorflow/graph/converter.py

```diff
@@ -274,47 +303,53 @@ def _get_nncf_graph_from_sequential(model: tf.keras.Model) -> NNCFGraph:
+        nncf_node = nncf_graph.add_nncf_node(layer_name,
+                                             node_type=layer_type,
+                                             node_metatype=layer_metatype,
+                                             module_attributes=module_attributes,
+                                             layer_name=layer_name,
+                                             is_shared=False)
+
+        if producer_layer_id is not None:
+            input_shape = _prepare_shape(model.get_layer(layer_name).input_shape)
+            nncf_graph.add_edge_between_nncf_nodes(producer_layer_id,
+                                                   nncf_node.node_id,
+                                                   tensor_shape=input_shape[0],
+                                                   input_port_id=0,
+                                                   dtype=Dtype.FLOAT)  # TODO: determine from keras layers
+        producer_layer_id = nncf_node.node_id
+
+    if layer_name is not None:
+        last_producer_layer_name = layer_name
+        last_producer_layer_id = producer_layer_id
+        output_model_layer = model.get_layer(last_producer_layer_name)
+        output_aux_node_name = PREFIX_AUXILIARY_OUTPUT_NODE + '_0'
+        output_node = nncf_graph.add_nncf_node(node_name=output_aux_node_name,
+                                               node_type=NNCFGraphNodeType.OUTPUT_NODE,
+                                               node_metatype=OutputNoopMetatype)
+        nncf_graph.add_edge_between_nncf_nodes(last_producer_layer_id, output_node.node_id,
+                                               tensor_shape=_prepare_shape(output_model_layer.output_shape),
+                                               input_port_id=0, dtype=Dtype.FLOAT)
 
     return nncf_graph
 
 
-def _get_module_attributes(layer: tf.keras.layers.Layer) -> ConvolutionModuleAttributes:
+def _get_module_attributes(layer: tf.keras.layers.Layer) -> ConvolutionLayerAttributes:
     channel_axis = get_input_channel_axis(layer)
     layer_ = unwrap_layer(layer)
     strides = layer_.strides[0]
     groups = layer_.groups
     kernel_size = layer_.kernel_size
 
-    return ConvolutionModuleAttributes(layer.trainable,
-                                       layer.get_input_shape_at(0)[channel_axis],
-                                       layer.get_output_shape_at(0)[channel_axis],
-                                       kernel_size,
-                                       strides,
-                                       groups)
+    return ConvolutionLayerAttributes(layer.trainable,
+                                      layer.get_input_shape_at(0)[channel_axis],
+                                      layer.get_output_shape_at(0)[channel_axis],
+                                      kernel_size,
+                                      strides,
+                                      groups, transpose=False)  # TODO: how to determine transpose here?
```

Comment:
Please, use OperatorMetatypes to resolve the TODO.

Reply:
Done